cluster/saltbase/salt/docker/init.sls

```diff
@@ -261,6 +261,20 @@ net.ipv4.ip_forward:
   sysctl.present:
     - value: 1
 
+{% if pillar.get('softlockup_panic', '').lower() == 'true' %}
```

Comment:
Another place is kubelet salt scripts. It doesn't matter at kubelet or docker salt script, it just need to run on every CVM node.